tests/sync/session/session.cpp

```diff
@@ -47,6 +47,7 @@ TEST_CASE("SyncSession: management by SyncUser", "[sync]") {
     if (!EventLoop::has_implementation())
         return;
 
+    reset_test_directory(tmp_dir());
```

Comment:
This is deleting /tmp or $TMPDIR which can do very bad things to any other running programs.